src/rating-menu/__tests__/__snapshots__/rating-menu.spec.ts.snap

```diff
@@ -60,8 +60,8 @@ exports[`StarRating should render with state 1`] = `
               
               <svg
                 aria-hidden="true"
-                class="ais-RatingMenu-starIcon"
-                ng-reflect-ng-class="ais-RatingMenu-starIcon"
+                class="ais-RatingMenu-starIcon ais-RatingMenu-starIcon--full"
+                ng-reflect-ng-class="ais-RatingMenu-starIcon ais-Ra"
```

Comment:
```
ng-reflect-ng-class="ais-RatingMenu-starIcon ais-Ra"
```

It ends with `ais-Ra`. Is this okay?
According to [this](https://stackoverflow.com/a/43473682/11593190), it's for debug purpose.

Reply:
Yes, I guess it's okay. I'm not sure why angular is cropping them that way but it's looks like it's a design choice. 
https://github.com/angular/angular/blob/f8096d499324cf0961f092944bbaedd05364eea1/packages/core/src/util/ng_reflect.ts#L24
see also https://github.com/angular/angular/issues/28786

Reply:
Thanks for the links :)